e2e_samples/parking_sensors_synapse/synapse/workspace/linkedService/sywsdevimgbb-WorkspaceDefaultStorage.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
You'll want to delete this file as it has workspace-specific (sywsdevimgbbb) which doesn't get replaced during the deployment script.